cli/src/hooks/next-check/index.ts

```diff
@@ -0,0 +1,21 @@
+import { Hook } from '@oclif/core';
+import { readFile } from 'fs/promises';
+import path from 'path';
+
+export const getSdkVersion = async (): Promise<null | string> => {
```

Comment:
I duplicated this code from compatibility.ts. It doesn't seem to run when imported from another file, I haven't figured out why.